src/LibYear/Output/PackageResult.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Text.Json.Serialization;
+using LibYear.Core;
+
+namespace LibYear.Output;
+
+internal sealed record PackageResult
+{
+	[JsonPropertyName("packageName")]
+	public string PackageName { get; set; } = string.Empty;
+	[JsonPropertyName("currentVersion")]
+	public DisplayVersion? CurrentVersion { get; init; }
+	[JsonPropertyName("latestVersion")]
+	public DisplayVersion? LatestVersion { get; init; }
+	[JsonConverter(typeof(DoubleFormatter))]
+	[JsonPropertyName("daysBehind")]
+	public double DaysBehind { get; init; }
```

Comment:
`DaysBehind` is simply an internal mechanism to help determine `YearsBehind` (libyears) and therefore shouldn't be exposed